admin/javascript/src/sections/campaign-admin/list.js

```diff
@@ -79,12 +81,44 @@ class CampaignListContainer extends SilverStripeComponent {
               {accordionGroups}
             </Accordion>
           </div>
+          <div className="cms-south-actions">
+            {this.renderButtonToolbar()}
+          </div>
         </div>
         { previewUrl && <CampaignPreview previewUrl={previewUrl} /> }
       </div>
     );
   }
 
+  renderButtonToolbar() {
+    const items = this.getItems(this.props.campaignId);
+    let itemSummaryLabel;
+
+    if (items) {
+      itemSummaryLabel = i18n.sprintf(
+       (items.length === 1) ?
+         i18n._t('Campaigns.ITEM_SUMMARY_SINGULAR')
+         : i18n._t('Campaigns.ITEM_SUMMARY_PLURAL'),
+       items.length
+     );
+
+      return (
+        <div className="btn-toolbar">
+          <FormAction
+            label={i18n._t('Campaigns.PUBLISHCAMPAIGN')}
+            style={'success'}
```

Comment:
This seems a bit weird. The `style` prop should take an object. See https://facebook.github.io/react/tips/inline-styles.html